### PR TITLE
Added abm labels

### PR DESCRIPTION
**Pull request by tenplus1**
Added labels to abm's so that code is easier to understand and mod makers can use them to search with in order to override abm's.

### CODE REVIEW

**Comment by sofar** [comment]
Does this work with the mod profiler? If so, then :+1:

File: mods/tnt/init.lua [hunk 1]
```diff
@@ -506,6 +506,7 @@ if enable_tnt then
 	})
 
 	minetest.register_abm({
+		label = "TNT Ignition",
```
Comment (paramat) [inline mods/tnt/init.lua]: ignition with small 'i' for consistency.

File: mods/fire/init.lua [hunk 2]
```diff
@@ -241,6 +242,7 @@ if minetest.setting_getbool("disable_fire") then
 	-- Remove basic flames only
 
 	minetest.register_abm({
+		label = "Remove flame",
```
Comment (paramat) [inline mods/fire/init.lua]: 'Remove disabled fire'

File: mods/fire/init.lua [hunk 3]
```diff
@@ -273,6 +276,7 @@ else
 	-- Remove flammable nodes
 
 	minetest.register_abm({
+		label = "Flame spread",
```
Comment (paramat) [inline mods/fire/init.lua]: 'Remove flammable nodes' this is not flame spread.

**Comment by paramat** [comment]
Line comments, otherwise +1.

**Comment by Fixer-007** [comment]
Yes, it will work with mod profiler.

**Comment by paramat** [comment]
:+1: And +1 from sofar.

**Comment by rubenwardy** [comment]
:+1: very nice

**Comment by paramat** [comment]
619ac5269393534c3f8ad8229514b17aa47eb513

**Pull request closed without merging**